cpu/msp430_common/startup.c

```diff
@@ -32,7 +33,7 @@ __attribute__((constructor)) static void startup(void)
 
     board_init();
 
-    puts("RIOT MSP430 hardware initialization complete.\n");
+    LOG_INFO("RIOT MSP430 hardware initialization complete.\n");
```

Comment:
This change removes the extra newline from the original code.
I don't have a board at hand to see how the output looks, but I guess that's ok.